samples/bluetooth/peripheral_hids_keyboard/src/main.c

```diff
@@ -234,22 +234,34 @@ static void caps_lock_handler(const struct bt_gatt_hids_rep *rep)
 }
 
 
-static void hids_outp_rep_handler(const struct bt_gatt_hids_rep *rep,
-				  struct bt_conn *conn)
+static void hids_outp_rep_handler(struct bt_gatt_hids_rep *rep,
+				  struct bt_conn *conn,
+				  bool write)
 {
 	char addr[BT_ADDR_LE_STR_LEN];
 
+	if (!write) {
+		printk("Output report read should not happen");
```

Comment:
Why shouldn't it happen?

I know this is quite unusual, but it is allowed by the spec. You can just state:
"Output report has been read"

Reply:
Missed "\n" character at the end of string?

Reply:
done